storybook/src/components/DateInput/DateInput.docs.mdx

```diff
@@ -12,10 +12,47 @@ import README from "../../../../packages/css/src/components/date-input/README.md
 
 <Controls />
 
-## Invalid
+## Examples
+
+### Date and time
+
+To let the user select a specific time on a date, use the `datetime-local` type.
+Note that the control’s UI varies in general from browser to browser.
+
+The value represents a local date and time, not necessarily the user’s local date and time.
+In other words, the input allows any valid combination of year, month, day, hour, and minute — even if such a combination is invalid in the user’s local time zone.
+
+<Canvas of={DateInputStories.DateTime} />
+
+### Invalid
+
+Indicates that the input value does not meet the specified constraints.
+The border of an invalid input is red.
```

Comment:
Do we need to describe this in text? It's already shown in the example

Reply:
I do intend to describe design choices more explicitly, but I’ve removed it for now.